docs/custom-flows/saml-connections.mdx

```diff
@@ -73,7 +71,6 @@ The following example shows two files:
       import { AuthenticateWithRedirectCallback } from '@clerk/nextjs'
 
       export default function SSOCallback() {
-        console.log('testing')
```

Comment:
I don't think we should have a `console.log` here, even if it's to debug at first, this can be added by the dev.